Src/Sarif.PatternMatcher/SharedUtilities.cs

```diff
@@ -9,9 +9,9 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher
 {
     public static class SharedUtilities
     {
-        public static void PopulateAssetFingerprint(string host, ref Fingerprint fingerprint)
+        public static void PopulateAssetFingerprint(string host, ref Fingerprint fingerprint, List<string> azureHosts = null)
```

Comment:
>PopulateAssetFingerprint [](http://example.com/codeflow?start=27&length=24)

can u add some tests to this function?

Reply:
i would also create a new method instead of this with:
list, host, out resource.

the list would not be null and it would be used  to the second method.

The second method would also receive:
list, host, out resource.

and this current method would hard-code the list.